optics-core/src/Optics.hs

```diff
@@ -109,15 +109,83 @@ module Optics
 
   -- * Optics utilities
 
-  -- ** Indexed optics
+  -- ** Re
 
-  -- | TODO
-  , module Optics.Indexed
+  -- | Some optics can be reversed with 're':
+  -- @'Iso' i s t a b@ into @'Iso' i b a t s@,
+  -- @'Getter' s t a b@ into @'Review' i b a t s@ etc.
+  -- Red arrows illustrate how 're' transforms optics:
```

Comment:
This is nice, thanks. The "Core definitions" section above still mentions `re`, even though the diagram no longer includes the red arrows.